integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/scaletest/TestReport.scala

```diff
@@ -16,16 +16,15 @@
 
 package com.nvidia.spark.rapids.tests.scaletest
 
-import java.io.{BufferedWriter, File, FileWriter}
-
+import java.io.{File, FileWriter}
+import org.apache.spark.sql.SparkSession
 import com.nvidia.spark.rapids.tests.scaletest.ScaleTest.Config
-
 /**
  * A Class for the report of Scale Test.
  * Only execution time are included at the beginning, will add more metadata for the test.
  * TODO: task failures, memory peak info, gpu usage etc.
  */
-class TestReport(config: Config, executionElapseMap: Map[String, Seq[Long]]) {
+class TestReport(config: Config, executionElapseMap: Map[String, Seq[Long]], spark:SparkSession) {
```

Comment:
Could we also have a section for errors while running? If you don't want to do it now, that is fine, but at least file a follow on for it.